### PR TITLE
Issue11 globalvars

### PR DESCRIPTION
**Pull request by NataniaBirnbaum**
I realized I made a mistake before, so I made a new pull request once I fixed it

### CODE REVIEW

File: PythonVisualizations/VisualizationApp.py [hunk 1]
```diff
@@ -67,23 +49,51 @@ def gridDict(frame):
...
-            self, 
-            window=None,     # Run visualization within given window
+
+class VisualizationApp(object):  # Base class for Python visualizations
+
+    # Default styles for display of values and operational controls
+    FONT_SIZE = 20
+    VALUE_FONT = ('Helvetica', FONT_SIZE)
+    VALUE_COLOR = 'black'
+    _FONT = ('Courier', FONT_SIZE * 8 // 10)
```
Comment (JMCanning78) [inline PythonVisualizations/VisualizationApp.py:69]: This looks like it should be `VARIABLE_FONT = ('Courier', FONT_SIZE * 8 // 10)`.  When I try to run Array.py, I get the error:
`AttributeError: 'Array' object has no attribute 'VARIABLE_FONT'`

File: PythonVisualizations/VisualizationApp.py [hunk 2]
```diff
@@ -67,23 +49,51 @@ def gridDict(frame):
...
+    VARIABLE_COLOR = 'brown3'
+    FOUND_FONT = ('Helvetica', FONT_SIZE)
+    FOUND_VARIABLECOLOR = 'green2'
```
Comment (lilypolonetsky) [inline PythonVisualizations/VisualizationApp.py:72]: I think this was meant to be `FOUND_COLOR`. That is what it was defined as before

File: PythonVisualizations/VisualizationApp.py (hunk 1 above)
Comment (lilypolonetsky) [inline PythonVisualizations/VisualizationApp.py:69]: I think this was meant to be `VARIABLE_FONT`

Comment (NataniaBirnbaum) [reply PythonVisualizations/VisualizationApp.py:69]: Whoops. I think I moved some text without realizing it.

**Pull request merged**